.devcontainer/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM rust:1.66
+
+RUN apt update && \
+    curl -fsSL https://deb.nodesource.com/setup_lts.x | bash - && \
+    apt update && \
+    apt install -y --no-install-recommends nodejs libssl-dev musl-dev make perl curl 
+
+RUN RUSTFLAGS=-Ctarget-feature=-crt-static cargo install wasm-pack cargo-chef \
```

Comment:
Either you don't need cargo-chef, or (better) you can steal the dependency pre-compilation step from this dockerfile: https://github.com/nitnelave/lldap/blob/main/Dockerfile#L25-L41

Reply:
I'll opt to remove chef. The container itself doesn't contain the code - vscode bind mounts it after creation